core/src/main/java/org/eclipse/hono/util/MessageHelper.java

```diff
@@ -494,6 +494,17 @@ public static String getCacheDirective(final Message msg) {
         return getApplicationProperty(msg.getApplicationProperties(), APP_PROPERTY_CACHE_CONTROL, String.class);
     }
 
+    /**
+     * Sets the time-to-live of the AMQP 1.0 message.
+     * 
+     * @param message the message for that the time-to-live is set.
+     * @param timeToLive The value of the property in seconds.
+     */
+    public static void setTimeToLive(final Message message, final long timeToLive) {
```

Comment:
I would prefer to use `Duration`, which makes it much easier to process IMHO.

Reply:
@ctron :+1: I have updated to use the Duration. Would you mind taking a look?